cmd/keeper/keeper.go

```diff
@@ -438,6 +438,47 @@ func (p *PostgresKeeper) updatePGState(pctx context.Context) {
 	p.lastPGState = pgState
 }
 
+// Since postgres 9.6
+//   https://www.postgresql.org/docs/9.6/static/runtime-config-replication.html
+// `synchronous_standby_names` can be in one of two formats:
+//   num_sync ( standby_name [, ...] )
+//   standby_name [, ...]
+// two examples for this:
+//   2 (node1,node2)
+//   node1,node2
+//
+// Versions only than 9.6 only support the second format.
+//
+// This function extracts the `[node1,node2]` part for both cases.
+func parse_synchronous_standby_names(s string) ([]string, error) {
+	var spacesSplit []string = strings.Split(s, " ")
+	var entries []string
+	if len(spacesSplit) < 2 {
+		// We're parsing format: standby_name [, ...]
+		entries = strings.Split(s, ",")
+	} else {
+		// We don't know yet which of the 2 formats we're parsing
+		_, err := strconv.Atoi(spacesSplit[0])
+		if err == nil {
+			// We're parsing format: num_sync ( standby_name [, ...] )
+			rest := strings.Join(spacesSplit[1:], " ")
+			inBrackets := strings.TrimSpace(rest)
+			if !strings.HasPrefix(inBrackets, "(") || !strings.HasSuffix(inBrackets, ")") {
+				return nil, fmt.Errorf("parse_synchronous_standby_names format has number but lacks brackets")
+			}
```

Comment:
I'll change the error to something like:
`synchronous standby string has number but lacks brackets`